nilearn/datasets.py

```diff
@@ -1066,6 +1066,38 @@ def fetch_smith_2009(data_dir=None, url=None, resume=True, verbose=1):
     return Bunch(**params)
 
 
+def fetch_atlas_power_2011(verbose=1):
+    """Download and load the Power et al. brain atlas composed of 264 ROIs.
+
+    Parameters
+    ----------
+    data_dir: string, optional
+        Path of the data directory. Used to force data storage in a non-
+        standard location. Default: None (meaning: default)
+    url: string, optional
+        Download URL of the dataset. Overwrite the default URL.
+
+    Returns
+    -------
+    data: sklearn.datasets.base.Bunch
+        dictionary-like object, contains:
+        - "rois": coordinates of 264 ROIs in MNI space
+
+
+    References
+    ----------
+    Power, Jonathan D., et al. "Functional network organization of the human
+    brain." Neuron 72.4 (2011): 665-678.
+    """
+    dataset_name = 'power_2011'
+    fdescr = _get_dataset_descr(dataset_name)
+    package_directory = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Not really for this PR, but it would feel safer to be based on nilearn.__file__ rather than __file__ in case this code is moved around in the package hierarchy.